packages/terra-core-docs/src/terra-dev-site/doc/form-input/About.1.doc.mdx

```diff
@@ -26,6 +27,12 @@ Generic input which represents an HTML input element directly.
 
 **Note:** To follow [UX Best Practices](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Placeholder_Research), this component does not allow `placeholder`.
 
+<Notice variant="caution" ariaLevel="3">
+
+**Accessibility Bug:** Currently a bug exists in Chromium based browsers with VoiceOver where `<input type="number" />` is read incorrectly when incrementing or decrementing the value with keyboard shortcuts. Terra Form Input utilizes `<input type="number" />` when given "number" as type, and thus will also have this screenreader issue. An alternative is to use something like `<input type="text" inputmode="numeric" pattern="\d*" />`, however this will have no built in validity checks or keyboard commands.
```

Comment:
```suggestion
**Accessibility Bug:** Due to a bug in Chromium-based browsers, VoiceOver would read `<input type="number" />` incorrectly when incrementing or decrementing the value with keyboard shortcuts. Terra Form Input utilizes `<input type="number" />` when the `type` prop is set to "number" and will also have this screenreader issue. An alternative is to use something like `<input type="text" inputmode="numeric" pattern="\d*" />`. However, this will have no built in validity checks or keyboard commands.
```

Reply:
Did you mean to use `input` or `TerraFormInput` here?
```js
<input type="text" inputmode="numeric" pattern="\d*" />
```

Reply:
```suggestion
**Accessibility Bug:** Currently a bug exists in Chromium based browsers (Chrome, Microsoft Edge, etc.) with VoiceOver where `<input type="number" />` is read incorrectly when incrementing or decrementing the value with keyboard shortcuts. Terra Form Input utilizes `<input type="number" />` when given "number" as type, and thus will also have this screenreader issue. An alternative is to use something like `<input type="text" inputmode="numeric" pattern="\d*" />`, however this will have no built-in validity checks or keyboard commands. Consumers are responsible for adding this functionality.
```

Reply:
> Did you mean to use `input` or `TerraFormInput` here?
> 
> ```js
> <input type="text" inputmode="numeric" pattern="\d*" />
> ```

I did mean to use `input`. While i think we would pass those attributes down, I don't know if we want them to use our terra-form-input in that way without a lot of testing.

Change to the paragraph here: [17557c3](https://github.com/cerner/terra-core/pull/3840/commits/17557c30e2583a402cc0755813e17bbd8ba26da6)